src/platform/msrCrypto/hash.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Use window crypto if it's available, otherwise use the msrCrypto module, else fall back to importing.
+const windowCrypto =
```

Comment:
Created this so that we have a common library to generate hashes, this is used in a number of places even in JS code.

Reply:
All of the hashing code eventually comes here, i.e. we now have a single way to generate hashes and this is the function that does that.

Reply:
This is a simple wrapper around the mscrypto js file,